Agents/PubChemAgent/pubchemagent/kgoperations/querytemplates.py

```diff
@@ -0,0 +1,408 @@
+
+def get_iri_query(inchi_string):
+    query="""
+    PREFIX os: <http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#>
+    PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+    SELECT ?speciesIRI ?Inchi
+    WHERE
+    {
+    ?speciesIRI rdf:type os:Species ;
+            os:inChI ?Inchi .
+      FILTER (str(?Inchi) ="#InChI#")
+    }    
+    """.replace('#InChI#', inchi_string)
```

Comment:
Rather than introducing these (horrific) placeholder strings and then replacing them with something else, it would be much preferable, more elegant, and recommended practice to use f-strings. Although I appreciate you probably won't be bothered changing the rather numerous occurrences of this... If you decide to do nothing about this at this stage, please add this as a to-do item in your main readme file.

Reply:
All the placeholder strings have been replaced with f-strings